### PR TITLE
Closes #13323 and carries

### PR DESCRIPTION
**Pull request by moxiegirl**
@cgwalters  et al. This carries your changes in the new structure.
Already approved on #13323 

Signed-off-by: Mary Anthony mary@docker.com

/cc @thaJeztah

### CODE REVIEW

File: daemon/graphdriver/devmapper/README.md [hunk 1]
```diff
@@ -71,204 +71,14 @@ status information about the driver.
...
-    and this option will just schedule device for deferred removal and
-    will not wait in a loop trying to remove a busy device.
-
-    Example use:
-
-    ``docker -d --storage-opt dm.use_deferred_removal=true``
-
+These options are currently documented both in [the man
+page](../../../man/docker.1.md) and in [the online
+documentation](http://docs.docker.com/reference/commandline/daemon/#docker-
```
Comment (thaJeztah) [inline daemon/graphdriver/devmapper/README.md]: can you change to `https://`?

File: docs/reference/commandline/daemon.md [hunk 2]
```diff
@@ -188,9 +205,9 @@ Currently supported options of `devicemapper`:
     10 GB of space on the pool. However, the filesystem will use more space for
     the empty case the larger the device is.
 
-     **Warning:** This value affects the system-wide "base" empty filesystem
-     that may already be initialized and inherited by pulled images. Typically,
-     a change to this value will require additional steps to take effect:
+    **Warning**: This value affects the system-wide "base" empty filesystem
```
Comment (thaJeztah) [inline docs/reference/commandline/daemon.md]: Do we want "warnings" to have the same look as "notes"? (feel free to ignore)

**Comment by thaJeztah** [comment]
thanks @moxiegirl. One minor note and a question. LGTM otherwise

File: docs/reference/commandline/daemon.md (hunk 2 above)
Comment (moxiegirl) [inline docs/reference/commandline/daemon.md]: Actually, this information isn't appropriate for a Warning or a Note.  It should just be part of the text --- which differed between the two. Sync'd. Thanks for the catch.

**Comment by moxiegirl** [comment]
Comments entered. Author LGTM.

**Comment by thaJeztah** [comment]
Thanks, @moxiegirl !

**Pull request merged**